manifest.yaml

```
@@ -1,8 +1,11 @@
 id: specter 
 title: Specter
-version: 2.0.2.2
-release-notes: | 
-  * Remove deprecated optional dependency of BTC RPC Proxy
+version: 2.1.1
+release-notes: |
+  * Updated to Specter Desktop v2.1.1
+  * Support for Bitcoin Core v28.x and v29.x
+  * Improved hardware wallet and multisig experience
+developer: "Alex71btc (https://github.com/Alex71btc)"
 license: mit
 wrapper-repo: https://github.com/Start9Labs/specter-wrapper
```

```suggestion
wrapper-repo: https://github.com/Alex71btc/specter-startos
```

you can go ahead and update this as well since you will be maintaining going forward